### DISCUSSION TITLE
Bottom safe area turn on

### DISCUSSION THREAD

**Discussion by quinn305** [LucasMucGH/BottomSheet#91] [Q&A] [answered]
I am trying to position the BottomSheet so it obeys the bottom safe area (I have tabs there), is there a way to do this yet?

**Comment by lucaszischka** [comment, marked as answer]
No this is not possible; you have 2 options:

1. Add padding at the bottom of your `mainContent`, so all of it is visible and reachable
2. Move the BottomSheet up in the view-hierarchy, so it is above the TabView. Now it overlays the TabView when shown and you don’t need to worry about the TabView. But please keep in mind, that while it is opened the TabBar is not accessible.
